### DISCUSSION TITLE
Themes? How/Where?

### DISCUSSION THREAD

**Discussion by 17500mph** [prasathmani/tinyfilemanager#551] [Q&A] [answered]
The information mentions 35+ themes. 
Are these for the file browser or the editor/viewer? 

.. and how do you invoke them? 
Or even make one yourself?

**Comment by prasathmani** [comment, marked as answer]
35+ themes are available only for ace editor, you can change theme color while editing the file. to configure the default theme #522 .
![image](https://user-images.githubusercontent.com/11345598/118609142-16e72c80-b7d8-11eb-9166-d3091f9d870a.png)

**Reply by 17500mph** [reply]
Gotcha, Thanks! 
For the editor, that's what I thought.. just wanted to make sure. 

...and see what I need to do so I can add support for other file types to the editor.

**Reply by prasathmani** [reply]
you can more file extension in `fm_get_text_exts()` function
![image](https://user-images.githubusercontent.com/11345598/118622845-4fd9ce00-b7e5-11eb-9199-e27c6c788109.png)
